doc/releases/changelog-dev.md

```diff
@@ -577,6 +577,10 @@
   optimization step updates.
   [(#1929)](https://github.com/PennyLaneAI/pennylane/pull/1929)
 
+* Fixes a bug where differentiating a QNode with multiple array
+  arguments via `qml.gradients.param_shift` throws an error.
+  [(#1989)]https://github.com/PennyLaneAI/pennylane/pull/1989
```

Comment:
I think you were missing some brackets here: 

```suggestion
  [(#1989)](https://github.com/PennyLaneAI/pennylane/pull/1989)
```

Reply:
Oh thanks 🤦